src/tests/base_test.py

```diff
@@ -826,16 +826,18 @@ def set_iptables_rules_for_nodes(
             cluster: Cluster,
             iptables_rules: List[IptableRule],
             given_nodes=None,
+            start_stop_nodes=True,
```

Comment:
when is this will ever be `False` ?

Reply:
The problem is that iptables fixture has no single responsibility and it handles shutdown nodes . basically we dont need to shutdown nodes to configure iptables .
The idea here is to apply iptables rules when node are running before installation /during installation

Trying to keep "old style" and allow  use iptables without touching the nodes and gain the cleanup method